MdeModulePkg/Bus/Pci/NvmExpressDxe/NvmExpress.c

```diff
@@ -1031,11 +1082,42 @@ NvmExpressDriverBindingStart (
       DEBUG ((DEBUG_WARN, "NvmExpressDriverBindingStart: failed to enable 64-bit DMA (%r)\n", Status));
     }
 
-    // MU_CHANGE - Support alternative hardware queue sizes in NVME driver
+    // MU_CHANGE [BEGIN] - Allocate IO Queue Buffer
+    //
+    // Set the Admin Queue Atttributes
+    //
+    Aqa = AllocateZeroPool (sizeof (NVME_AQA));

```

Comment:
Memory Leak on Aqa?

 Its allocated and used, but it never appears to be freed, and it seems to go out of scope at the end of the call.

Reply:
Good call, thanks! Made it a local variable.